src/mysql/connection.cr

```diff
@@ -64,7 +87,7 @@ class MySql::Connection < DB::Connection
   end
 
   # :nodoc:
-  def read_ok_or_err
```

Comment:
These formatter changes are unrelated. I opened a separate PR for that: https://github.com/crystal-lang/crystal-mysql/pull/116